src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -191,6 +191,22 @@
             </i>
           </div>
         </div>
+
+        <div class="form-group row"
+             *ngIf="data.applications.selected.includes('rbd')">
+          <label i18n
+                 class="cd-col-form-label"
```

Comment:
Just to align them
```suggestion
                 class="cd-col-form-label pt-0"
```

Reply:
why is this @cloudbehl? It doesn't look nice with the rest of the spacing. so I'd say to remove this and keep the default behavior. @aaSharma14 
![Screenshot from 2024-06-05 10-59-41](https://github.com/ceph/ceph/assets/71764184/bc94014f-fe77-433a-8b5b-a20cf0c97ea5)

Reply:
btw, https://github.com/ceph/ceph/pull/57755#pullrequestreview-2098102003

Reply:
If we don't do this, the checkbox goes off. This is consistent with RGW form as well.

Reply:
i was complaining about the `pt-0`. And it is not really consistent with RGW, because there we have a sub heading called Encryption and then an Enable label with the checkbox. And that's also not consistent with the rest of the pages though.

Reply:
oh now i get it. you mean checkbox goes off.. hmm, then its actually a problem with the checkbox, we should modify the style of the checkbox rather than the label.. because now the label is off

Reply:
and also if you check the carbon design system they have the label on the other side of the checkbox, not like this.